src/marqo/tensor_search/api.py

```diff
@@ -191,6 +192,12 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.post('/validation')
```

Comment:
`validate/index` is better. Can add other validation endpoints with this format later